pkg/build/gobuild_test.go

```diff
@@ -766,12 +771,32 @@ func TestGoBuild(t *testing.T) {
 		want := map[string]string{
 			"foo":   "bar",
 			"hello": "world",
+			"baz":   "qux",
 		}
 		got := cfg.Config.Labels
 		if d := cmp.Diff(got, want); d != "" {
 			t.Fatalf("Labels diff (-got,+want): %s", d)
 		}
 	})
+
+	t.Run("check annotations", func(t *testing.T) {
+		m, err := img.Manifest()
+		if err != nil {
+			t.Fatalf("Manifest() = %v", err)
+		}
+
+		want := map[string]string{
+			"qux": "quux",
+		}
+		got := m.Annotations
+
+		if d := cmp.Diff(got, want, cmp.FilterPath(func(p cmp.Path) bool {
```

Comment:
Can you just add these to `want` ?

Reply:
We could, but `org.opencontainers.image.base.digest` changes to random value for each test run. 🤷‍♂️ So, had to filter it out. Any ideas?